site/profile/manifests/windows/registry.pp

```diff
@@ -0,0 +1,43 @@
+class profile::windows::registry
+(
+  String $ntpserver = "time.windows.com",
+)
+
+{
+
+#Puppet module for managing base Windows registry settings
+
+# Disable UAC
+  registry_value { 'HKLM\Software\Microsoft\Windows\CurrentVersion\Policies\System\EnableLUA':
+    ensure => 'present',
+    data   => ['0'],
+    type   => 'dword',
+  }
+
+# Configure NTP Server
+  registry_value { 'HKLM\SYSTEM\Currentcontrolset\Services\W32time\Parameters\NtpServer':
+    ensure => 'present',
+    data   => ["$ntpserver,0x9"],
+    type   => 'string',
+  }
+
+# Disable Windows Firewall
```

Comment:
This is a demo, so usually the more examples we can show, the better. If you disagree and would be happier with just the one, that's fine too. Just wanted to build a bit more meat into the examples.